srcpkgs/uriparser/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'uriparser'
+pkgname=uriparser
+version=0.8.6
+revision=1
+wrksrc=${pkgname}-${version}
```

Comment:
this is the default value of `wrksrc`, so you can remove this line

Reply:
Fixed.